lib/Config.php

```diff
@@ -153,6 +159,28 @@ public function isRecordingEnabled(): bool {
 		return $isSignalingOk && $recordingEnabled;
 	}
 
+	public function getRecordingFolder(string $userId): string {
+		return $this->config->getUserValue(
+			$userId,
+			'spreed',
+			'recording_folder',
+			$this->getAttachmentFolder($userId) . '/Recording'
+		);
+	}
+
+	public function getRecordingAllowedMimes(): array {
+		$allowedMimes = $this->config->getAppValue(
+			'spreed',
+			'allowed_recording_mimes',
+			json_encode(self::DEFAULT_ALLOWED_RECORDING_FORMATS)
+		);
+		$allowedMimes = json_decode($allowedMimes, true);
+		if (is_array($allowedMimes) && count($allowedMimes)) {
```

Comment:
**Optional:** If you don't need the number, use empty instead of count (faster)
```suggestion
		if (empty($allowedMimes) && is_array($allowedMimes)) {
```